### PR TITLE
py-custom_inherit: Submission

### PR DESCRIPTION
**Pull request by essandess**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: python/py-custom_inherit/Portfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+homepage            https://github.com/rsokl/custom_inherit
+
+checksums           rmd160  0aafd9e709150525a8cc9fc4ce47c01b948f564c \
+                    sha256  7052eb337bcce83551815264391cc4efc2bf70b295a3c52aba64f1ab57c3a8a2 \
+                    size    31193
+
+python.versions     38 39 310 311
+
+if {${name} ne ${subport}} {
+    python.pep517   yes
```
Comment (reneeotten) [inline python/py-custom_inherit/Portfile:24]: this can/should be moved outside of the `if` statement that becomes then obsolete

**Review by reneeotten — CHANGES_REQUESTED** [review]

File: python/py-custom_inherit/Portfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-custom_inherit
+version             2.4.1
+platforms           darwin
```
[the remark this answers is not in the document]
Comment (essandess) [reply python/py-custom_inherit/Portfile]: Thanks, done.

File: python/py-custom_inherit/Portfile [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-custom_inherit
+version             2.4.1
```
[the remark this answers is not in the document]
Comment (essandess) [reply python/py-custom_inherit/Portfile:7]: Thanks, done.

File: python/py-custom_inherit/Portfile (hunk 1 above)
Comment (essandess) [reply python/py-custom_inherit/Portfile]: Thanks, done.

**Review by reneeotten — APPROVED** [review]

**Pull request merged**